src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -74,6 +75,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case FindCommand.COMMAND_WORD:
             return new FindCommandParser().parse(arguments);
 
+        case FindTagCommand.COMMAND_WORD: case FindTagCommand.COMMAND_ALIAS:
```

Comment:
same here: support for COMMAND_ALIAS has been dropped
you can remove the match for COMMAND_ALIAS switch case